dist/hint.js

```diff
@@ -19,23 +19,23 @@ angular.hint.onAny(function (data, severity) {
 
 },{"./loader.js":2,"angular-hint":3}],2:[function(require,module,exports){
 /**
- * @license AngularJS v1.3.0-build.3042+sha.76e57a7
- * (c) 2010-2014 Google, Inc. http://angularjs.org
+ * @license AngularJS v1.6.4-build.5322+sha.d96e58f
+ * (c) 2010-2017 Google, Inc. http://angularjs.org
```

Comment:
It is modified in a couple of places (e.g. `minErr` doesn't have access to `toDebugString()` so it uses a simpler implementation and there are a couple of other changes (marked with `ANGULAR HINT ALTERATION` comments) to support 1.2.x as well.

Maybe it is better to remove the header and leave a comment that it is based on v1.6.4 loader.js?